src/log-analytics/HISTORY.rst

```diff
@@ -1,12 +1,22 @@
+0.1.4
+++++++++++++++++++
```

Comment:
Shouldn't there be only one history.rst change?

Reply:
Yes, PyPi.

Reply:
https://pypi.org/manage/project/log-analytics/releases/ see past releases

Reply:
Ah I see. Thanks for explaining. I think next time it is better to have the PR merged in before publishing the whl file on pypi.